audio_service/lib/audio_service.dart

```diff
@@ -421,7 +444,7 @@ class MediaItem {
 
   /// Creates a [MediaItem].
   ///
-  /// [id], [album] and [title] must not be null, and [id] must be unique for
+  /// The [id], [album] and [title] must not be null, and [id] must be unique for
```

Comment:
Just thinking about it, it's now redundant to say in the documentation certain values must not be null. The type system now speaks for itself.

Reply:
i'm wondering, why album and title must be unique per instance?

Reply:
They don't have to be, only `id` must be unique. All three must be "not null", and it's the "not null" but that can now be removed from the docs as it's implied already by the types.

Reply:
oh sorry i misread this